content/doc/book/pipeline/pipeline-best-practices.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: section
+title: Pipeline Best Practices
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author: Alex Taylor
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+:toc:
+endif::[]
+
+Pipeline Best Practices:
+
+Q: How do I scale pipelines so that they are not using too many resources on the master?
```

Comment:
Do we want it in the Question/answer format or a sub-title within `Pipeline Best Practices`? e.g. `Scaling Jenkins master` and this list set of things to do to achieve this?